src/reducers/cars.js

```diff
@@ -4,15 +4,15 @@ const initalStore = {
       name: 'Ford Ranger 3.2 TDCi Doppelkabine Limited',
       description: null,
       stockNumber: 'JP84315',
-      auctionEndTime: null,
+      auctionEndTime: new Date(Date.now() + 300000),
```

Comment:
antd is using moment for dates, we might have some problems down the road

Reply:
but I do not use this filed with antd

Reply:
probably will be used if we save the car from the form to redux